javers-persistence-sql/src/main/java/org/javers/repository/sql/schema/FixedSchemaFactory.java

```diff
@@ -39,6 +39,7 @@
     public static final String SNAPSHOT_COMMIT_FK =    "commit_fk";
     public static final String SNAPSHOT_GLOBAL_ID_FK = "global_id_fk";
     public static final String SNAPSHOT_TYPE =         "type";
+    public static final String VERSION =               "version";
```

Comment:
I don't know whether it will be slow, since I don't have that much experience with really large databases, but your proposal sounds good and reasonable :)
Two more questions - do you have any other comments to be fixed and the one I asked in the previous comment - which is the field that is the same for all snapshots of a given entity.